tmt/steps/provision/podman.py

```diff
@@ -283,6 +284,33 @@ def _run_ansible(
             log=log,
             silent=silent)
 
+    @functools.cached_property
+    def _is_toolbox(self) -> bool:
+        """ Return ``True`` if running in toolbox, ``False`` otherwise. """
+        if os.path.exists('/run/.toolboxenv'):
```

Comment:
`os.path` should be replaced with `Path().exists()`

Reply:
need to remember we use pathlib here :)

Reply:
moved to facts, where we run shell commands, so this does not apply anymore, resolving.